sentry-types/src/protocol/profile.rs

```diff
@@ -1,77 +1,143 @@
+use std::collections::HashMap;
+
 use super::v7::{DebugMeta, TraceId};
+
+use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize, Serializer};
 use uuid::Uuid;
 
 fn serialize_id<S: Serializer>(uuid: &Uuid, serializer: S) -> Result<S::Ok, S::Error> {
     serializer.serialize_some(&uuid.as_simple())
 }
+#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
+/// Metadata about the transaction associated with the profile
+pub struct TransactionMetadata {
+    #[serde(serialize_with = "serialize_id")]
+    /// Transaction ID
+    pub id: Uuid,
+    /// Transaction Name
+    pub name: String,
+    /// Trace ID
+    pub trace_id: TraceId,
+    /// Transaction start timestamp in nanoseconds
+    pub relative_start_ns: u64,
```

Comment:
relative to what?

Reply:
good point, let me add that in the comments!